src/web-api.js

```diff
@@ -7,446 +7,594 @@ const Signature = require('./signature');
 const Utilities = require('./utilities');
 const IDApi = require('./id-api');
 const { mapServerUri, sdkVersionInfo, validatePartnerParams } = require('./helpers');
-
-class WebApi {
-  constructor(partner_id, default_callback, api_key, sid_server) {
-    this.partner_id = partner_id;
-    this.default_callback = default_callback;
-    this.sid_server = sid_server;
-    this.api_key = api_key;
-    this.url = mapServerUri(sid_server);
+const { getWebToken } = require('./web-token');
+
+/**
+ * Validates if the information required to submit a job is present.
+ *
+ * @param {{
+ *  entered: boolean|string|undefined,
+ *  country: string|undefined,
+ *  id_type: string|undefined,
+ *  id_number: string|undefined,
+ * }} idInfo - ID information required to create a job.
+ * @param {number} jobType - Smile Job Type
+ * @returns {string} value representing if `entered` is true or false.
+ */
+const validateIdInfo = (idInfo, jobType) => {
+  if (!('entered' in idInfo) || idInfo.entered.toString() === 'false') {
+    if (jobType === 6) { // NOTE: document verification does not check for `country` and `id_type`.
+      ['country', 'id_type'].forEach((key) => {
```

Comment:
Will this check for country and id_type? The comment seems to denote that this will not be checked for document verification